pkg/providers/vsphere/config/template-cp.yaml

```diff
@@ -1043,8 +1043,8 @@ stringData:
       name: cloud-provider-vsphere-credentials
       namespace: kube-system
     stringData:
-      {{.vsphereServer}}.password: "{{.eksaVspherePassword}}"
-      {{.vsphereServer}}.username: "{{.eksaVsphereUsername}}"
+      {{.vsphereServer}}.password: "{{(or .eksaControlPlanePassword .eksaVspherePassword)}}"
```

Comment:
Is there a plan to migrate to 1 way of doing this? In general, 1 way is better than 2 but with env variables I feel like its more of a necessity as it can be confusing whats configuring what.

Reply:
At some point this should be modified to use something other than environment variables.